tests/components/element-card/element-card.spec.tsx

```diff
@@ -0,0 +1,105 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { ElementCard } from "src/components";
+import { EElementType, TElement } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper, mockBuilderState } from "src/util/test-helper";
+
+const mockElement = {
+    internalId: "mock123",
+    type: EElementType.EMAIL,
+    id: "mockElement",
+    required: false,
+    label: ELEMENT_BUTTON_LABELS[EElementType.EMAIL],
+};
+
+const mockOnClick = jest.fn();
```

Comment:
- move the mocks to the bottom of the file
- put them under a `MOCKS` comment divider
- change `mockElement` to `MOCK_ELEMENT` since it is a const declared at the root level